docs/docs-beta/docs/dagster-plus/deployment/code-locations/code-location-history.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Code location history"
```

Comment:
This doesn't need to be merge-blocking, but I think we could get a more actionable/useful title here.

Ideally a user would be able to tell from the sidebar and title that this is where they'd go to rollback if they have a problem. wdyt, @cmpadden? I'm struggling to come up with something that isn't a mouthful.

Reply:
But in that case, what does `deployment` refer to? Rolling back an entire full deployment? We use that term in a number of ways, unfortunately :|

Reply:
Good point.

"image rollback", "version rollback", "Code location history and image rollback"

You're right that they're a mouthful, hah.

Reply:
I updated to "Code location history and rollbacks" -- it's a bit vague but at least it gives people who explicitly are looking for rollbacks a breadcrumb